project.clj

```diff
@@ -28,27 +30,34 @@
                          [net.logstash.logback/logstash-logback-encoder "4.7"]
                          [org.codehaus.janino/janino "2.7.8"]
                          [com.fasterxml.jackson.core/jackson-core "2.7.3"]
+                         [org.yaml/snakeyaml "1.13"]
```

Comment:
Yeah, the yaml one here was to resolve a conflict between puppetserver/jruby and code manager.

As far the criteria for inclusion in this file, I'd say it's basically:

* Things that will be used by more than one or two projects, and
* Things that are necessary to resolve transitive dependency conflicts between multiple PL projects.